node/node.d.ts

```diff
@@ -965,16 +965,37 @@ declare module "child_process" {
         stdio?: any;
         customFds?: any;
         env?: any;
+        encoding: void;
```

Comment:
https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback
encoding is optional.
We need to allow omission of the argument.